starfyre/parser.py

```diff
@@ -167,18 +164,23 @@ def handle_endtag(self, tag):
             if len(self.stack) > 0:
                 parent_node = self.stack[-1]      #this is last item/"top element" of stack
 
-                if endtag_node.original_name != "": #this node is for "custom" component, so we should check for <slot> tags 
-                    #process endtag_node children
+                if endtag_node.original_name != endtag_node.tag: #this node is for "custom" component
+                    #process endtag_node children                    
                     new_children = []
-                    for  child_component in endtag_node.children:
+                    is_slot_used = False
+                    for child_component in endtag_node.children:
                         if child_component.tag == "slot":
                             new_children.extend(self.current_children)
+                            is_slot_used = True
                         else:
                             new_children.append(child_component)
+                    if not is_slot_used and len(self.current_children) > 0:
+                        new_children.extend(self.current_children)
+                        print("Appending at the end of the stack as slot position not specified.")
```

Comment:
I thought we discussed not appending the new children. Just printing saying not appending anything

Reply:
> I thought we discussed not appending the new children. Just printing saying not appending anything

@sansyrox 

Actually no :sweat_smile: . Last week the discussion was to append it to the end in case `<slot> tag` is not provided and give feedback to the user. But I can change it if you would prefer :hugging: